### DISCUSSION TITLE
Primer sprint.

### DISCUSSION THREAD

**Discussion by albsierra** [2DAW-CarlosIII/webMarcaPersonalFPEstudiantes#1] [Ideas]
Para el desarrollo del proyecto por parte de nuestros alumnos, es conveniente que se empiecen ya a dar los primeros pasos. Estos primeros pasos, en el caso de servidor, incluyen:

- La preparación del entorno de la aplicación.
- El conocimiento de los casos de uso.
- El esquema de bases de datos que dará soporte a la aplicación.
- El desarrollo de las primeras vistas y el boceto del layout.

En este último punto, intervendrían los conocimientos de **Marcas** y **Diseño**.
Hay que tener en cuenta que este primer _sprint_ de Servidor, cuando lo traslademos al curso 2023/24, yo lo enmarco en el _sprint_ correspondiente a las 2 primeras semanas de noviembre.

Esta tarea la he planificado en el [proyecto](https://github.com/orgs/2DAW-CarlosIII/projects/1) en el que veremos el avance y desarrollo de la web. Esa tarea es la denominada **Creación de la aplicación Laravel** y los detalles de la misma están en un enlace al AulaVirtual. Posteriormente, creo que convendría que estas tareas las fuéramos publicando en el repositorio de GitHub del proyecto, pero será otro día.

Un saludo a tod@s y mucho ánimo.

**Comment by albsierra** [comment]
@fortiz79 @carmenmquz @victorgarri @AndresEsparzaALU @JaimeLloretGarcia @VicenteJavier para poder centralizar la discusión sobre el desarrollo de este proyecto, debéis hacer click en `watch` en el repositorio [webMarcaPersonalFPEstudiantes](https://github.com/2DAW-CarlosIII/webMarcaPersonalFPEstudiantes).

El primero que reciba el mensaje, por favor, que comente en este hilo, para saber que los mensajes se están recibiendo correctamente.

**Reply by AndresEsparzaALU** [reply]
Recibido!

**Reply by carmenmquz** [reply]
¡Listo!

**Comment by JaimeLloretGarcia** [comment]
Hecho!
